Backend/src/main/java/com/pg/programmerground/controller/PlaygroundController.java

```diff
@@ -22,38 +26,55 @@
      * 무한 스크롤 적용위한 방법? 찾기
      */
     @GetMapping("")
-    public ResponseEntity<ApiResponse<List<PlaygroundCardInfoDto>>> playgroundList() {
+    public ResponseEntity<ApiResponse<List<PlaygroundCardResponse>>> playgroundList() {
         return ResponseEntity.ok().body(new ApiResponse<>(playgroundService.getPlaygroundCardList()));
     }
 
     /**
      * playground 생성
      */
     @PostMapping("")
-    public ResponseEntity<ApiResponse<Long>> makePlayground(@Valid @RequestBody MakePlaygroundInfoDto info) throws Exception {
+    public ResponseEntity<ApiResponse<Long>> createPlayground(@Valid @RequestBody PlaygroundApi info) throws Exception {
         return ResponseEntity.ok().body(new ApiResponse<>(playgroundService.createPlayground(info)));
     }
 
     /**
      * playground 참여 신청
      */
     @PostMapping("/apply/{playgroundId}")
-    public ResponseEntity<ApiResponse<Boolean>> applyPlayground(@PathVariable Long playgroundId, @RequestBody ApplyPlaygroundDto applyPlayground) throws Exception {
+    public ResponseEntity<ApiResponse<Boolean>> applyPlayground(@PathVariable Long playgroundId, @RequestBody ApplyPlaygroundApi applyPlayground) throws Exception {
```

Comment:
```suggestion
    public ResponseEntity<ApiResponse<PlaygroundAplyResultResponse>> applyPlayground(@PathVariable Long playgroundId, @RequestBody ApplyPlaygroundApi applyPlayground) throws Exception {
```

오브젝트로 일관적으로 리턴하는게 클라이언트에서 처리하기 편함